src/JSON.js

```diff
@@ -0,0 +1,23 @@
+
+export default function findPath(obj, value) {
+
+  let result = []
+
+  console.log('value', value)
+  console.log('obj', obj)
+
+  for(let key in obj) {
+    console.log('======>', key)
+    if(obj[key] == value) {
+      result = `${key} -> `
+      result.push(findPath(value))
+    } else {
+      result.push(value)
```

Comment:
The `value` is not supposed to be pushed to the final result, just the _path_ to the specified `value`.  Otherwise, implementation looks good!